game/requesthandler.cc

```diff
@@ -169,7 +170,13 @@ void RequestHandler::Post(web::http::http_request request)
         if(!songPointer) {
             request.reply(web::http::status_codes::NotFound, "Song \"" + utility::conversions::to_utf8string(jsonPostBody[utility::conversions::to_string_t("Artist")].as_string()) + " - " + utility::conversions::to_utf8string(jsonPostBody[utility::conversions::to_string_t("Title")].as_string()) + "\" was not found.");
             return;
-        } else {
+        }
+        else if (songPointer->loadStatus == Song::LoadStatus::ERROR)
+        {
+            request.reply(web::http::status_codes::NotFound, "Song \"" + utility::conversions::to_utf8string(jsonPostBody[utility::conversions::to_string_t("Artist")].as_string()) + " - " + utility::conversions::to_utf8string(jsonPostBody[utility::conversions::to_string_t("Title")].as_string()) + "\" Song load status is error. Please check what's wrong with it.");
```

Comment:
Would be good to break the line in to smaller pieces.

Reply:
Fixed